app-backend/db/src/main/scala/com/azavea/rf/database/SceneToProjectDao.scala

```diff
@@ -147,8 +147,13 @@ object SceneToProjectDao extends Dao[SceneToProject] with LazyLogging {
       countO map {
         (count: Int) => logger.debug(s"Using ${stpsWithFootprints.length} scenes in project out of $count")
       }
+      
       val stps = stpsWithFootprints map { _._1 }
-      val md = MosaicDefinition.fromScenesToProjects(stps)
+      
+      val md = compositeOption match {
+        case Some((redBand, greenBand, blueBand)) => MosaicDefinition.fromScenesToProjects(stps, redBand, greenBand, blueBand)
```

Comment:
These are named here, and that's good, but I wonder whether it wouldn't be clearer to have the bands as separate
`Option[Int]` arguments and then use cats to get the thing you want here, e.g.:

```scala
 @ val x: Option[Int] = Some(3) 
x: Option[Int] = Some(3)

@ val y: Option[Int] = None 
val zy: Option[Int] = None

@ val z: Option[Int] = Some(4) 
z: Option[Int] = Some(4)

@ (x, y, z).tupled 
res4: Option[(Int, Int, Int)] = None

@ (x, z, x).tupled 
res5: Option[(Int, Int, Int)] = Some((3, 4, 3))
```

I fear things where we have to keep track of the order on our own.

Reply:
Neat. I'll make that change.